### PR TITLE
Update Stake.Com.fc

### PR DESCRIPTION
**Pull request by aviramdayan**
update-time-and-more

### CODE REVIEW

File: Stake.Com/Stake.Com.fc [hunk 1]
```diff
@@ -622,47 +610,15 @@ public class stake : FaucetScript
             if (IsVisible(ReCaptchaIframe))
             {	
 		      Log("call to solve captch");
-              var result = DoSolveCaptcha(); //call to solve captch		   
+              var result = DoSolveCaptcha(); //call to solve captch		 
+		      ClaimSuccess(); //Marks the claim as a success
```
Comment (faucetcollector) [inline Stake.Com/Stake.Com.fc:614]: Is it always a success here?

File: Stake.Com/Stake.Com.fc [hunk 2]
```diff
@@ -512,6 +510,36 @@ public class stake : FaucetScript
...
+        {
+			
+           var MinVar = ElementByXPath("//article[contains(@class,'maincontent')]//span[2]/span[1]") ;
+		   var SecVar = ElementByXPath("//article[contains(@class,'maincontent')]//span[3]/span[1]") ;
+
+           if (IsVisible(MinVar))
+           {
+	         Log("ClaimBTC - WaitTimerText apper - return to site time"); 
+             int min,sec;
+             min = Convert.ToInt32(MinVar.Text.Trim().Replace("m.", "").Replace("s.", "").Split(' ')[0])*60;
```
Comment (faucetcollector) [inline Stake.Com/Stake.Com.fc:527]: You know what I am gonna comment when I see this

Comment (aviramdayan) [reply Stake.Com/Stake.Com.fc:527]: hehe :) yeah, enough, i like this method..

File: Stake.Com/Stake.Com.fc (hunk 1 above)
Comment (aviramdayan) [reply Stake.Com/Stake.Com.fc:614]: Yes if the text does not appear, then it will always be a success.

**Pull request merged**